packages/import/tests/schema/import-schema.spec.ts

```diff
@@ -476,7 +479,7 @@ describe('importSchema', () => {
 
       directive @foo on FIELD_DEFINITION
 
-      extend type User @key(fields: "id") {
+      extend type User @key(fields: "id") @key(fields: "id") {
```

Comment:
Should we really repeat the same directive definitions as in here?

Reply:
You're right. We should be removing the duplicates.

I've adjusted the logic to account for this. Ty.